src/Subscription.php

```diff
@@ -612,6 +612,28 @@ public function getCurrencyAttribute()
         return optional($this->plan())->amount()->getCurrency()->getCode();
     }
 
+    /**
+     * Gets the amount to be refunded for the subscription's unused time.
+     *
+     * @param  \Carbon\Carbon|null  $now
+     * @return \Money\Money
+     */
+    public function getReimburseAmountForUnusedTime(?Carbon $now = null): ?Money
+    {
+        $now = $now ?: now();
+
+        if ($this->onTrial()) {
+            return null;
```

Comment:
How about we return a zero Money amount instead of null? Feels more consistent. We're also doing this on `reimbursableAmount()`